Restructure docs (#55)
cbrnr/sleepecg: doc/source/api.rst

SleepECG API Reference
======================

.. automodule:: sleepecg

> This page gives an overview of all public SleepECG functions and classes.

Reviewer: ```suggestion
This page lists all public SleepECG functions and classes.
```
